cmd/livepeer_cli/wizard_stats.go

```diff
@@ -197,8 +197,8 @@ func (w *wizard) orchestratorStats() {
 		[]string{"Price Per Segment", eth.FormatUnits(t.PricePerSegment, "ETH")},
```

Comment:
Can remove this now?

Reply:
fixed in https://github.com/livepeer/go-livepeer/pull/993/commits/54863dce0b699a81e16eb27e09bf64b6b28e79a8